histomicstk/features/FeatureExtraction.py

```diff
@@ -60,6 +64,7 @@ def FeatureExtraction(Label, In, Ic, K=128, Fs=6, Delta=8):
         - MeanGradMag, StdGradMag, SkewnessGradMag, KurtosisGradMag,
         - EntropyGradMag, EnergyGradMag,
         - SumCanny, MeanCanny
+
```

Comment:
@cooperlab @cdeepakroy I think we need to update docstring in the refactored FeatureExtraction.py